app/src/services/census/CensusStreamService.ts

```diff
@@ -17,13 +18,28 @@ export default class CensusStreamService implements ServiceInterface {
 
     private readonly config: Census;
 
+    private readonly lastMessagesMap: Map<string, Date> = new Map<string, Date>();
```

Comment:
Maybe use `Date.now()`, would assume numbers are a bit better performance/memory wise.

Reply:
You can do that in a type declaration?

Reply:
No, you just use number as the type and then instead of new Date use `Date.now()`.